firstvoices/backend/models/signals/visibility_signals.py

```diff
@@ -0,0 +1,27 @@
+from django.db import transaction
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from backend.models.jobs import BulkVisibilityJob, JobStatus
+from backend.tasks.visibility_tasks import bulk_visibility_change_job
+from firstvoices.celery import link_error_handler
+
+
+@receiver(post_save, sender=BulkVisibilityJob)
+def request_bulk_visibility_change(sender, instance, **kwargs):
+    """
+    Starts the bulk visibility change job when a BulkVisibilityJob is created.
+    """
+    if BulkVisibilityJob.objects.filter(
+        status=JobStatus.STARTED, site=instance.site
+    ).exists():
```

Comment:
A detail about the timing of this check: we need to check for running jobs when we start running the bulk_visibility_change_job task, not when we queue it.